web/src/settings_org.js

```diff
@@ -1142,6 +1222,20 @@ function enable_or_disable_save_button($subsection_elem) {
             break;
         }
     }
+
+    return disable_save_btn;
+}
+
+function enable_or_disable_save_button($subsection_elem) {
+    const time_limit_settings = [...$subsection_elem.find(".time-limit-setting")];
+
+    let disable_save_btn = false;
+    if (time_limit_settings.length) {
+        disable_save_btn = should_disable_save_button_for_time_limit_settings(time_limit_settings);
+    } else if (is_video_chat_provider_jitsi_meet()) {
+        disable_save_btn = should_disable_save_button_for_jitsi_server_url_setting();
```

Comment:
This should be called only for the relevant subsection. Currently this disables the save button for other subsections as well.